server/pom.xml

```diff
@@ -82,14 +82,14 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>ditaa</groupId>
+      <groupId>io.kroki.thirdparty.ditaa</groupId>
       <artifactId>ditaa</artifactId>
-      <version>${ditaa.version}</version>
+      <version>${project.version}</version>
```

Comment:
Is it possible to keep a "fixed" version? If not I guess that we can remove the reference in the `<properties>` right?

Reply:
Not if you want to use the Maven release tooling...

Reply:
I'm pretty sure there's a good reason but I'm not sure I understand why... we are using a "released" version no? Sorry, I'm sure I'm not using the correct terminology but I would expect this dependency to be resolved as a "standard" dependency.
But I guess the `thirdparty/ditaa/pom.xml` is not really the pom of the dependency but the pom of the project that contains a jar. So we cannot define `<version>1.3.13</version>` in `thirdparty/ditaa/pom.xml` and use it in here?

Anyway, I think it's fine, I'm mainly asking to learn 🤓

Reply:
Once you get the release plugin automating the version number bumps you’ll need all the poms to align their version number. 

If you’re not using *a* release plugin and not using `-SNAPSHOT` versions on master then you could have different versions